server/webserver.go

```diff
@@ -231,12 +231,12 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 			return
 		}
 
-		ok, err := s.LivepeerNode.Eth.CurrentRoundLocked()
+		isLocked, err := s.LivepeerNode.Eth.CurrentRoundLocked()
 		if err != nil {
 			respondWith500(w, err.Error())
 			return
 		}
-		if !ok {
```

Comment:
Since this would have been something a test would've caught what do you think about adding a test case that a) fails without this change and b) passes with this change?

Reply:
Added in e8e0bb51 along with some form param tests , decided to omit some other stuff would take quite a bit of time to add 100% coverage due to all the several codepaths and is not relevant for this PR.

Reply:
Not sure why the subsequent tests fail all of a sudden still, I made sure to properly close the server and request bodies in 133b459

Reply:
> Not sure why the subsequent tests fail all of a sudden still

915349c should fix those tests. Previously, e8e0bb5 passed `eth.StubClient{}` as an arg to the LivepeerNode constructor in `newMockServer()` which caused test cases in `TestGetEthChainID` that expected LivepeerNode to be in off-chain mode (the test cases for error cases) to break.